packages/session-cohttp-async/session-cohttp-async.0.4.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+name: "session"
```

Comment:
The `name` fields in these PRs are wrong; they can just be removed as they are inferred in the main repository.